tests/lib/systems.sh

```diff
@@ -27,3 +34,28 @@ is_ubuntu_14_system(){
     fi
     return 1
 }
+
+get_snap_for_system(){
+    snap=$1
+    system=${2:-$SPREAD_SYSTEM}
+
+    if [[ "$system" == ubuntu-core-18-* ]]; then
```

Comment:
Perhaps use a switch statement here, it's a natural fit for this kind of code.

Reply:
Totally agree, thanks for the review